src/main/java/com/example/eventorganizer/registration/RegistrationService.java

```diff
@@ -0,0 +1,28 @@
+package com.example.eventorganizer.registration;
+
+import com.example.eventorganizer.user.User;
+import com.example.eventorganizer.user.UserRepository;
+import com.example.eventorganizer.user.UserRole;
+import lombok.AllArgsConstructor;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class RegistrationService {
+    private final UserRepository userRepository;
+    private final BCryptPasswordEncoder bCryptPasswordEncoder;
+
+    public String register(RegistrationRequest request){
+        User user = new User(
+                request.getFirstName(),
+                request.getLastName(),
+                request.getEmail(),
+                bCryptPasswordEncoder.encode(request.getPassword()),
+                UserRole.USER
+        );
+
+        userRepository.save(user);
+        return "Registration successful";
```

Comment:
Please return id or user returned from reporitory.save instead of custom string. You can display that message ion frontend when you receive 200 status response.